chatwoot/chatwoot
# feat: Add additional attributes in `setUser` method

Fixes #3599 

#### How Has This Been Tested?

- Give values `company_name`, `location` and `description` in messenger script

Ex:
```
window.addEventListener('chatwoot:ready', function() {
  window.$chatwoot.setUser('<%= user_id %>', {
    email: 'jane@example.com',
    description:'', // description about the user
    country_code:'' // Two letter country code
    city: '' // City of the user
    social_profiles:{
     twitter:'', // Twitter user name 
     linkedin:'', // LinkedIn user name 
     facebook:'' , // Facebook user name 
   }
  });
})
```
- Load the widget

- Check attributes are showing in conversation side panel
